doc/README.build_env.md

```diff
@@ -63,8 +63,9 @@ usage: build_env.py [-h] [--conda_build_config CONDA_BUILD_CONFIG]
                     [--repository_folder REPOSITORY_FOLDER]
```

Comment:
I'd created an issue to update this documentation for conda environment files that are generated. And have also been working on it. With this PR, I'll need to update that again.